config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  namespace :api do
+    namespace :v1 do
+      resource :patients, only: [] do
```

Comment:
This feels weird. Why not `namespace :patients` here if you aren't using any resources?

Reply:
Fixed in aa7a18d8b98d3d400cb2b081e3e3a5ab2a3ea315.